spec/models/category_spec.rb

```diff
@@ -11,7 +11,7 @@
 
   it 'validates uniqueness of name' do
     Fabricate(:category)
-    is_expected.to validate_uniqueness_of(:name).scoped_to(:parent_category_id)
+    is_expected.to validate_uniqueness_of(:name).scoped_to(:parent_category_id).case_insensitive
```

Comment:
[Unexpected failure when using case_insensitive and scoped_to](https://github.com/thoughtbot/shoulda-matchers/issues/1092)

Specifically this [comment](https://github.com/thoughtbot/shoulda-matchers/issues/1092#issuecomment-378864653)